manifests/server.pp

```diff
@@ -21,7 +21,7 @@
     if $::puppet::server_git_repo {
       $config_version_cmd = "git --git-dir ${::puppet::server_envs_dir}/\$environment/.git describe --all --long"
     } else {
-      $config_version_cmd = ''
+      $config_version_cmd = undef
```

Comment:
I'm slightly concerned about this, plus using `if scope.lookupvar(..)` in the template, as setting a variable to undef sometimes results in :undef symbols and things in ERB templates with lookupvar.  Should we make it a local variable in `puppet::server::config` to avoid lookupvar?